android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMKeyboard.java

```diff
@@ -135,10 +137,10 @@ public void initKMKeyboard(final Context context) {
 
     setWebChromeClient(new WebChromeClient() {
       public boolean onConsoleMessage(ConsoleMessage cm) {
+        String msg = String.format("KMW JS Log: Line %d, %s:%s", cm.lineNumber(), cm.sourceId(), cm.message());
         if (KMManager.isDebugMode()) {
-          Log.d("KMEA", "Keyman JS Log: Line " + cm.lineNumber() + ", " + cm.sourceId() + ":" + cm.message());
           if (cm.messageLevel() == ConsoleMessage.MessageLevel.ERROR) {
-            Log.e("KMEA", "Keyman JS Log: ERROR");
+            Log.d(TAG, msg);
```

Comment:
I lowered this level to `Log.d` because the Log.e gets sent to Sentry in `sendKMWError()` (line 153)